tests/mock/Concrete/PlatformOrderDecorator.php

```diff
@@ -0,0 +1,899 @@
+                $additionalInformation["cc_savecard_{$index}"] === '1';
+
+            $creditCardDataIndex = AbstractCreditCardPayment::getBaseCode();
+            if (!isset($paymentData[$creditCardDataIndex])) {
+                $paymentData[$creditCardDataIndex] = [];
+            }
+            $paymentData[$creditCardDataIndex][] = $newPaymentData;
+        }
+    }
+
+    private function extractMultibuyerData(
+        $prefix,
+        $additionalInformation,
+        $index = null
+    )
+    {
+        $index = $index !== null ? '_' . $index : null;
+
+        if (
+            !isset($additionalInformation["{$prefix}_buyer_checkbox{$index}"]) ||
+            $additionalInformation["{$prefix}_buyer_checkbox{$index}"] !== "1"
+        ) {
+            return null;
+        }
+
+        $fields = [
+            "{$prefix}_buyer_name{$index}" => "name",
+            "{$prefix}_buyer_email{$index}" => "email",
+            "{$prefix}_buyer_document{$index}" => "document",
+            "{$prefix}_buyer_street_title{$index}" => "street",
+            "{$prefix}_buyer_street_number{$index}" => "number",
+            "{$prefix}_buyer_neighborhood{$index}" => "neighborhood",
+            "{$prefix}_buyer_street_complement{$index}" => "complement",
+            "{$prefix}_buyer_city{$index}" => "city",
+            "{$prefix}_buyer_state{$index}" => "state",
+            "{$prefix}_buyer_zipcode{$index}" => "zipCode",
+            "{$prefix}_buyer_home_phone{$index}" => "homePhone",
+            "{$prefix}_buyer_mobile_phone{$index}" => "mobilePhone"
+        ];
+
+        $multibuyer = new \stdClass();
+
+        foreach ($fields as $key => $attribute) {
+            $value = $additionalInformation[$key];
+
+            if ($attribute === 'document' || $attribute === 'zipCode') {
+                $value = preg_replace(
+                    '/\D/',
+                    '',
+                    $value
+                );
+            }
+
+            $multibuyer->$attribute = $value;
+        }
+
+        return $multibuyer;
+    }
+
+    private function extractPaymentDataFromMundipaggBilletCreditcard(
+        $additionalInformation,
+        &$paymentData, $payment
+    )
+    {
+        $moneyService = new MoneyService();
+        $identifier = null;
+        $customerId = null;
+
+        $brand = null;
+        try {
+            $brand = strtolower($additionalInformation['cc_type']);
+        } catch (\Throwable $e) {
+
+        }
+
+        if (isset($additionalInformation['cc_token_credit_card'])) {
+            $identifier = $additionalInformation['cc_token_credit_card'];
+        }
+
+        if (
+            !empty($additionalInformation['cc_saved_card']) &&
+            $additionalInformation['cc_saved_card'] !== null
+        ) {
+            $identifier = null;
+        }
+
+        if ($identifier === null) {
+            $objectManager = ObjectManager::getInstance();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'Mundipagg\Core\Test\Mock\Concrete\ObjectManager' in method 'extractPaymentDataFromMundipaggBilletCreditcard'.](https://app.codacy.com/manual/mundipagg/ecommerce-module-core/pullRequest?prid=4926694)